### PR TITLE
TECH Bump i18n, lodash, update circle to validate against node 6,7,8

### PR DESCRIPTION
**Pull request by PerrinStephane**
### Jira issue:
https://transcovo.atlassian.net/browse/TRAN-
or
https://transcovo.atlassian.net/browse/TECH

### Description:
Bump i18n, lodash, update circle to validate against node 6,7,8

### Coverage result:
```
=============================== Coverage summary ===============================
Statements   :
Branches     :
Functions    :
Lines        :
================================================================================
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,25 +9,26 @@
...
   "devDependencies": {
     "chai": "~3.5.0",
     "co-mocha": "~1.1.2",
     "coveralls": "~2.11.6",
     "eslint": "~1.10.3",
-    "eslint-config-cp": "transcovo/eslint-config-cp#1.1.0",
+    "eslint-config-cp": "github:transcovo/eslint-config-cp#1.1.0",
     "express": "^4.14.0",
     "istanbul": "~0.4.2",
+    "is-path-in-cwd": "^2.0.0",
```
Comment (lledey) [inline package.json]: What is this for? 🤔

File: package.json [hunk 2]
```diff
@@ -46,7 +47,7 @@
     "url": "git+https://github.com/transcovo/express-middlewares.git"
   },
   "engines": {
-    "node": "8.10.0",
+    "node": ">=6.0.0",
     "npm": ">=2.14.7"
```
Comment (lledey) [inline package.json]: You should remove this and let the npm version depend on the node version installed

File: package-lock.json [hunk 3]
```diff
@@ -1,28 +1,46 @@
 {
   "name": "express-middleware",
-  "version": "1.9.5",
+  "version": "2.0.0",
```
Comment (lledey) [inline package-lock.json]: You should completely remove this file as npm coming with node 6 will not update it

File: package.json (hunk 1 above)
Comment (PerrinStephane) [reply package.json]: ... needed to run test locally

File: package-lock.json (hunk 3 above)
Comment (PerrinStephane) [reply package-lock.json]: It's usefull for newest version of node

Comment (lledey) [reply package-lock.json]: There will be issues, we use node 6 locally and when installing a dependency this file will become out of date and lead to unexpected dependencies versions installed in clients

File: package.json (hunk 2 above)
Comment (PerrinStephane) [reply package.json]: ok

**Pull request merged**